virtual-server/examples/kubectl/virtual-server-shared-pvc.yaml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: your-pv-claim
+  namespace: my-namespace
+spec:
+  accessModes:
+  - ReadWriteMany
+  storageClassName: shared-nvme-ord1
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 20Gi
+---
+apiVersion: virtualservers.coreweave.com/v1alpha1
+kind: VirtualServer
+metadata:
+  name: example-vs-shared-pvc
+  namespace: my-namespace
+spec:
+  region: ORD1
+  os:
+    type: linux
+  resources:
+    gpu:
+      type: Quadro_RTX_4000
+      count: 1
+    cpu:
+      count: 3
+    memory: 16Gi
+  storage:
+    root:
+      size: 40Gi
+      storageClassName: block-nvme-ord1
+      source:
+        pvc:
+          namespace: vd-images
+          name: ubuntu2004-docker-master-20210323-ord1
+    additionalDisks:
```

Comment:
`additionalDisks` is for block volumes. `filesystems` is for shared filesystems (via VirtioFS)